collection_manager/collection_manager/services/CollectionProcessor.py

```diff
@@ -59,13 +59,13 @@ async def process_granule(self, granule: str, collection: Collection):
                         f"'{collection.dataset_id}'.")
             use_priority = collection.historical_priority
         else:
-            logger.info(f"Granule '{granule}' detected but has already been ingested or is not in desired "
-                        f"time range for collection '{collection.dataset_id}'. Skipping.")
+            logger.debug(f"Granule '{granule}' detected but has already been ingested or is not in desired "
+                         f"time range for collection '{collection.dataset_id}'. Skipping.")
```

Comment:
Since we don't manage time range any more for a collection you can remove the second page of the log message: everything after "or is not in ..." can be removed from message.

Reply:
good point, will do